doc-content/enterprise-only/project-data/migrating-legacy-designer-proc.adoc

```diff
@@ -0,0 +1,41 @@
+[id='migrating-from-legacy-designer-proc']
+
+= Migrating business processes to the new designer
+
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. Existing users that intend to utilize the functionality in the new designer should start migrating their processes to the new designer. All new processes should be created using the new process designer.
```

Comment:
Ok, I really hate 'utilize the functionality' :-) use the functions of the new....
and for consistency should it be 'new process designer' always? I think it should because that component doesn't have a real name, so the least we can do it give it the respect it deserves in a three-word phrase. That's a real question because I maintain the BPMN reference. Something to consider.

There are a lot of 'should's here. Just sayin.
should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate

Consider rewriting:
 Existing users that intend to utilize the functionality in the new designer should start migrating their processes to the new designer.

If you intend to use the new process designer, start migrating your processes to the new designer. Create all new processes in the new process designer.

Note: Maybe one day someone will give this component a name, or if not, maybe in 7.5 we can drop the 'new' and just call it process designer.

Reply:
I agree with all suggested changes and have made the same. Yes, the new process designer is what I will use consistently as that is what I continue to hear from dev up to now.